### ISSUE TITLE
Spam in two channels/servers at once

### ISSUE THREAD

**Issue by BeemoBeemo** [ExtraConcentratedJuice/pokecord-catcher#175]
Is it possible to spam in two different servers at once? I can fill out one channel ID, but if i do [channel id1, channel id2], for example it doesnt work. Is there a way to write it down in the config so it does? 

This will increase spawnrate - so beneficial
Thanks

**Comment by Swearinbag** [comment]
just copy the folder x amount of times, make  script that runs all 4 in one go- I got it to work with gnome-terminal and now they are running on my raspberry pi :)

**Comment by BeemoBeemo** [comment]
Hey man, thanks for the reply it helped. With what script can I use to make them all run at at once? Currently I have to open a command prompt for each folder (bot) separately and manually- is there a way to just run every single one with one command? You mentioned gnome terminal but there are probably some specific scripts.. thanks!

**Comment by Swearinbag** [comment]
yep, mine run all at once just different tabs. just search how to run scripts with gnome-terminal and how to open a new tab too! from there just copy paste :D

**Comment by BeemoBeemo** [comment]
Thanks for the help! I'm like a 100% coding noob, so I honestly have no idea how to make it work (even starting gnome terminal is too much for me haha) I copied this script, but I don't know what to fill in where.

ForEach ($folder in (Get-ChildItem -Path test -Directory)) { dotnet test $folder.FullName }

**Comment by BeemoBeemo** [comment]
Okay so basically the script only works when I launch cmd or powershell from the subdirectory itself (right click on directory > open powershell > type dotnet pokecordcatcher.dll).

When I launch this from a standard powershell or cmd:

ForEach ($folder in (“the path to the folder containing the .dll file”)) { dotnet Pokecordcatcher.dll }

it says the regular:

Did you mean to run dotnet SDK commands? Please install dotnet SDK from:
  http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409

I don't understand why it doesn't work?

**Comment by Swearinbag** [comment]
Problem is is that I'm running on Linux with "gnome terminal" which I have no clue how to do on windows but the way I fixed my issue with that one is to make sure that the described dotnet version is installed from the original post and to make sure that the dll file is typed with no caps mistakes. As far as I remember it should be "dotnet PokecordCatcher.dll" with a captial C.

**Comment by BeemoBeemo** [comment]
Hmm unfortunately it didn't work. I guess I'll try to stick to opening them all individually- Gnome terminal is a bit above understanding having never done this kind of stuff before. Thank you for your help anyway! 

If you or someone else figures it out however please contact me, much appreciated~
